src/application/widget.rs

```diff
@@ -422,6 +408,7 @@ impl relm::Widget for Widget {
             pref_popover: gtk::Popover::new(None::<&gtk::Button>),
             defered_button: gtk::CheckButton::with_label("Display defered tasks"),
             done_button: gtk::CheckButton::with_label("Display done tasks"),
+            environment: env,
```

Comment:
It’s better if the variable have the same name of the struct field, it’s should be shortened.

Reply:
Done!